### DISCUSSION TITLE
2006/11/17/%E3%80%8A%E7%99%BE%E5%AE%B6%E8%AE%B2%E5%9D%9B%E3%80%8B%E8%8B%8F%E8%BD%BC

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#130] [Announcements]
# 2006/11/17/%E3%80%8A%E7%99%BE%E5%AE%B6%E8%AE%B2%E5%9D%9B%E3%80%8B%E8%8B%8F%E8%BD%BC

到昨天，《百家讲坛》10集的苏轼专题刚好讲完。《百家讲坛》是我最喜欢的电视节目之一，不过，从我开始收看它起，它的主体几乎都是历史题材的，可能是历史人物和故事最容易被大众所接受吧。到目前为止，我最喜欢的系列讲座是《明亡清兴60年》，究其原因，主要是因为袁崇焕是一个我比较钦佩的历史人物。只可惜这个主题是断断续续播出的，中间插了一些其它话题，到现在也还没有完全播完。

https://blog.qizhen.xyz/2006/11/17/%E3%80%8A%E7%99%BE%E5%AE%B6%E8%AE%B2%E5%9D%9B%E3%80%8B%E8%8B%8F%E8%BD%BC

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Rubin
2008年07月22日 10:05 编辑
     整个人类的自由，不感奢望，人类历经数千载，喜怒哀乐始终绕着我们左右，这是构成生活的因素。
     一个人的自由是可以追求的，是心灵上的自由。但要修到庄子的境界，逍遥的境界。

回复
三叶草
2006年11月24日 09:23 编辑
不知什么时候被你加为好友的，呵，真是不好意思，我今天才知道！赶紧跑过来，是个很不错的空间！！只是计算机相关的有些深啊，我不太看得懂，呵，虽然我自己也是学计算机，但感觉学到的只是皮毛。这篇我是看明白了，哈，不明白就太哀了，我也很敬仰苏轼，那个百家讲坛我没看过，听你这么一说，我是一定要看了！！呵呵！

回复
竹贤
2006年11月20日 09:50 编辑
我们都是凡夫俗子，我们都食人间烟火，我们都有喜怒哀乐，这很正常。要始终保持乐观豁达的情怀，需要我们一辈子的努力，一辈子的修养。

回复
Moca
2006年11月19日 09:03 编辑
谢谢帮我复习了一边

回复
Mavis
2006年11月18日 23:26 编辑
苏轼是个全才，勇敢打破词的韵律限定，才出现以后许多传世名篇。。。。。。
他曾调侃别人，一树梨花压海棠。。。。。。哈哈哈哈~~~~~

回复
yoyo
2006年11月17日 17:50 编辑
呵呵 感概很多么！

回复
永青
2006年11月17日 17:14 编辑
这首词和苏大学士的小妾有关吧。。。

回复
carrie
2006年11月17日 17:13 编辑
两首江城子，感触颇深。
十年生死两茫茫，不思量，自难忘。
千里孤坟无处话凄凉。
纵使相逢应不识，尘满面，鬓如霜。 夜来幽梦忽还乡。小轩窗，正梳妆。 相顾无言，惟有泪千行。 料得年年肠断处：明月夜，短松冈。

回复
